src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -40,6 +40,25 @@ void RotatedSPOs::createRotationIndices(int nel, int nmo, RotationIndices& rot_i
       rot_indices.emplace_back(i, j);
 }
 
+void RotatedSPOs::createRotationIndicesFull(int nel, int nmo, RotationIndices& rot_indices)
+{
+  // start with core-active rotations - put them at the beginning of the list
+  // so it matches the other list of rotation indices
+  for (int i = 0; i < nel; i++)
+    for (int j = nel; j < nmo; j++)
+      rot_indices.emplace_back(i, j);
+
+  // Add core-core rotations - put them at the end of the list
+  for (int i = 0; i < nel; i++)
+    for (int j = i + 1; j < nel; j++)
+      rot_indices.emplace_back(i, j);
+
+  // Add active-active rotations - put them at the end of the list
+  for (int i = nel; i < nmo; i++)
+    for (int j = i + 1; j < nmo; j++)
+      rot_indices.emplace_back(i, j);
```

Comment:
Is it possible to precompute the size of rot_indices and reserve it? I feel nmo * nel scale emplace_back can be quite costly due to resize.

Reply:
This function isn't going to run that frequently, and the resize algorithm should double the capacity every time it runs out, so this really shouldn't be a big issue.

But yes, the size can be precomputed, and it should be nmo*(nmo-1)/2.